client/extensions/woocommerce/woocommerce-services/views/carrier-accounts/settings.js

```diff
@@ -148,7 +148,7 @@ export const CarrierAccountSettings = ( props ) => {
 					<p className="carrier-accounts__settings-subheader-description">
 						{ translate(
 							'Set up your own UPS carrier account to compare rates and print labels from multiple carriers in WooCommerce Shipping. Learn more about adding {{a}}carrier accounts{{/a}}.',
-							{ components: { a: <a href="https://docs.woocommerce.com/document/using-your-own-ups-account-in-woocommerce-shipping/" /> } }
+							{ components: { a: <a href="https://docs.woocommerce.com/document/productid-type-permalinks/using-your-own-ups-account-in-woocommerce-shipping//" /> } }
```

Comment:
There is an extra `/` at the end. The link works though. :D

Reply:
Whoopsie, thanks! 💯 
```suggestion
							{ components: { a: <a href="https://docs.woocommerce.com/document/productid-type-permalinks/using-your-own-ups-account-in-woocommerce-shipping/" /> } }
```